modules/hitlnext/src/backend/workspace.ts

```diff
@@ -20,6 +20,10 @@ const ROLE_CONFIGURATION = [
         res: `module.${MODULE_NAME}`,
         op: '+r+w'
       },
+      {
+        res: `module.${MODULE_NAME}.resolve`,
+        op: '-r-w'
```

Comment:
Wouldn't those ops remove read and write access for agents to resolve handoffs?

Reply:
What we check on the endpoint that allows resolving handoffs is this:

`hasPermissionOrAgentOnline('resolve', 'write'),`

So from my understanding, this is how we'll call `hasPermission`:

`await bp.http.hasPermission(req, 'write', 'module.hitlnext.resolve', true)`

and when I check the agent role you edited, you don't give him write access to this resource. What am I missing here?

Reply:
1 -
```
if (hasPermission) {
  return next()
} else {
  return agentOnlineMiddleware(req, res, next)
}
```

2 - 
``` 
   {
     res: `module.${MODULE_NAME}`,
      op: '+r+w'
   },
```

At reference 1: First, we check if the user has permission to resolve (This is a new kind of permission and should be given to API users, but the current block (reference 2) also allows it since it's a child). If not, check if it's an agent and is online (Same middleware as before). I had to remove the permission because every agent would not need to be online to resolve (since the user would have permission), as required by default